Source: upstream git, https://trac.bunkus.org/ticket/941
Upstream: yes
Reason: generate png files, otherwise an empty directory appears

From 03011e08f99491a55df0ce5498f31414f912045f Mon Sep 17 00:00:00 2001
From: Moritz Bunkus <mosu@sweet-chili.local.bunkus.org>
Date: Sun, 15 Dec 2013 12:42:43 +0100
Subject: [PATCH] build system: install PNG images for the guides as well

Fixes #941.
---
 Rakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Rakefile b/Rakefile
index 47066ad..5455bce 100755
--- a/Rakefile
+++ b/Rakefile
@@ -531,7 +531,7 @@ namespace :install do
 
       $languages[:guides].each do |language|
         install_data "#{c(:docdir)}/guide/#{language}/",        FileList[ "#{$top_srcdir}/doc/guide/#{language}/mkvmerge-gui.*" ]
-        install_data "#{c(:docdir)}/guide/#{language}/images/", FileList[ "#{$top_srcdir}/doc/guide/#{language}/images/*.gif"   ]
+        install_data "#{c(:docdir)}/guide/#{language}/images/", FileList[ "#{$top_srcdir}/doc/guide/#{language}/images/*.{gif,png}" ]
       end
     end
   end
-- 
1.8.5.1

